faqs/galaxy/analysis_reporting_issues.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Reporting Issues and Feature Requests
+area: analysis
+box_type: tip
+layout: faq
+contributors: [jennaj, garimavs]
+---
+
+- For reporting **Usage Problems**, related to tools and functions, head to the [Galaxy Help](https://help.galaxyproject.org/) site.
+    - **Red error datasets**:
+        - To resolve it you might be asked to send a bug report.
+        - Click on the green bug icon within one of the red error datasets to start the submission process 
+    - **Green unexpected-result datasets**:
+        - To resolve it you may be asked to send in a shared history link and possibly a shared workflow link. 
```

Comment:
The easiest is probably to directly to make the history accessible via link. Even if not needed this would avoid exchange of several emails.

Reply:
So should I not mention the email part and keep only the shared History (via link) thing?

Reply:
Okay, maybe @jennaj can help us here. I will mail you both the content and we can then look for improvements.

Reply:
however you phrase it here, please link to the existing FAQ about how to share a history via link